beps/0001-notifications-system/README.md

```diff
@@ -160,89 +168,202 @@ The following backend service interfaces are added as part of this proposal.
 ```ts
 // TODO - We may want to add an additional wrapping here with interfaces for Notification and NotificationParameters
 
-interface SendNotificationRequest {
-  receivers: SignalReceivers;
+export type NotificationRecipients =
+  | {
+      type: 'entity';
+      entityRef: string | string[]; // typically a user or group entity, otherwise `spec.owner` of the entity/entities
+    }
+  | {
+      type: 'broadcast'; // all users
+    };
 
-  /* TODO - please contribute :) */
-}
+export type NotificationType = 'undone' | 'done' | 'saved';
 
-interface SendNotificationResponse {
+export type Notification = {
   id: string;
-  receivers: SignalReceivers;
+  userRef: string;
+  title: string;
+  description: string;
+  origin: string;
+  link: string;
+  created: Date;
+  saved: boolean;
```

Comment:
What is the behavior behind `saved`?

Reply:
Users can save notifications for later. This is in use for example in the GitHub notifications and makes a lot of sense in UX point of view

Reply:
Got it. It's up to FE to give it the right naming. We can have `pinned`. Or `stared` (like on slack). Or "saved for later". The semantics is similar.

Reply:
Yeah, we can call it what feels the right 👍 I think you might have some good ideas for the UI so I hope we can get the PR merged soonish and you can re-design the whole thing, what do you think?